src/firefly/plans/xafs_scan.py

```diff
@@ -338,9 +338,10 @@ def reset_default_regions(self):
         self.ui.regions_spin_box.setValue(default_num_regions)
 
         # set default values for testing, to be deleted in the future
```

Comment:
Are these still default values for testing? Or do we want these to be the production default values?

Reply:
I think these default values are useful, let's make it production default value?

Reply:
I agree. Can we update the comment on line 340 then?

Reply:
will do